config/cable.yml

```diff
@@ -1,6 +1,6 @@
 development:
   adapter: redis
```

Comment:
maybe we can use `inline` ?

Reply:
According to [Action Cable docs](https://guides.rubyonrails.org/action_cable_overview.html#dependencies), _The default adapter in new Rails applications is the asynchronous (async) adapter_.
WDYT if we use `async` instead? Assuming Hotwire will still working 😅

Reply:
yes sorry for the confusion, I meant that one haha 😂